tests/media_element.rs

```diff
@@ -0,0 +1,25 @@
+use web_audio_api::context::{AudioContext, AudioContextOptions, BaseAudioContext};
+use web_audio_api::node::AudioNode;
+use web_audio_api::MediaElement;
+
+#[test]
+fn test_media_element_source_progress() {
+    let options = AudioContextOptions {
+        sink_id: "none".into(),
+        ..AudioContextOptions::default()
+    };
+    let context = AudioContext::new(options);
+
+    let mut media = MediaElement::new("samples/major-scale.ogg").unwrap();
+    media.set_loop(true);
+
+    let src = context.create_media_element_source(&mut media);
+    src.connect(&context.destination());
+    media.play();
+
+    // sleep for a bit, make sure the audio thread has started
```

Comment:
Please add a `TODO` or `FIXME` comment that allows to find and resolve these little workarounds or hacks later.